demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -93,6 +94,8 @@ def is_valid_file(self, validate_rn: bool = True, skip_test_conf: bool = False)
             self.is_valid_image(),
             self.is_valid_description(beta_integration=False),
             self.is_valid_max_fetch_and_first_fetch(),
+            self.is_valid_entity_description_no_escape_sequences(),
```

Comment:
```suggestion
            self.is_valid_integration_description(),
```

Leave room for additional check in the description in the future.

Reply:
done